Makefile

```diff
@@ -176,6 +186,12 @@ out/minikube_$(DEB_VERSION).deb: out/minikube-linux-amd64
 	dpkg-deb --build out/minikube_$(DEB_VERSION)
 	rm -rf out/minikube_$(DEB_VERSION)
 
+out/minikube-all-amd64.tar.gz: pkg/minikube/assets/assets.go out/localkube out/minikube-linux-amd64 out/minikube-darwin-amd64 out/minikube-windows-amd64.exe
+	tar czvf out/minikube-all-amd64.tar.gz $($(shell echo TAR_TARGETS_ALL | tr a-z A-Z))
+
+out/minikube-%-amd64.tar.gz: pkg/minikube/assets/assets.go out/minikube-$(GOOS)-$(GOARCH)$(IS_EXE)
```

Comment:
Calling MAKE inside the tar file should work for now for the targets.  I have updated the make rule.